src/users/users.service.ts

```diff
@@ -0,0 +1,39 @@
+import { PrismaService } from '@/prisma/prisma.service';
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { Users } from '@prisma/client';
+import { isNil } from 'lodash';
+
+@Injectable()
+export class UsersService {
+	constructor(private readonly prismaService: PrismaService) {}
+
+	// email認証
```

Comment:
```suggestion
	/**
	 * email認証処理
	 * @throws {paths["/api/v1/users/password-reset"]["post"]["responses"]["400"]}
	 */
```

修正不要、こういう感じだと嬉しいのかな
<img width="611" alt="image" src="https://github.com/ih14c-13-14/artist-backend/assets/49822810/309c8468-353c-47b0-8636-db27487ac2a6">